rollup.config.js

```diff
@@ -0,0 +1,32 @@
+import { getBabelOutputPlugin } from '@rollup/plugin-babel';
+import { terser } from 'rollup-plugin-terser';
+
+export default {
+  input: 'lib/ical/module.js',
+  output: [
+    { file: 'dist/ical.js',  format: 'es', exports: "default" },
+    {
+      file: 'dist/ical.min.js',
+      format: 'es',
+      exports: "default",
+      plugins: [terser()]
+    },
+    {
+      file: 'dist/ical.es5.js',
+      exports: "default",
+      format: 'cjs',
+      plugins: [
+        getBabelOutputPlugin({ presets: ['@babel/preset-env'] })
+      ],
+    },
+    {
+      file: 'dist/ical.es5.min.js',
+      exports: "default",
+      format: 'cjs',
+      plugins: [
```

Comment:
package.js (NodeJS) has a concept of “Conditional exports” — https://nodejs.org/api/packages.html#conditional-exports . It can utilize the right input for `import ` (ES6 modules) / `require()` (ES5 CJS), depending on how the user utilizes the library.  ical.js would benefit by offering “Conditional exports” of the ES6/ES5 output.

Reply:
Good point. I don't think I've tested using this from node as a package extensively.